tox.ini

```diff
@@ -141,6 +135,11 @@ show_missing = False
 count = False
 max-line-length = 88
 extend-ignore = E203
+docstring-convention = numpy
+per-file-ignores =
+    tests/*:D
+    benchmarks/*:D
```

Comment:
Is it introducing a new behaviour? Or is it a migration from something from before?

Reply:
It's migration of the old behaviour, but slightly disguised. What these options do is disabling `pydocstyle`-related warnings (`DXXX`) for tests and benchmarks, since we don't write docstrings for them.

Previously we just didn't run `pydocstyle` on these two folders, but now we have to because it's part of `flake8`.

So all in all, this reproduces the old behaviour.